### ISSUE TITLE
mozillaGradingWorker always seems to return an A grade

### ISSUE THREAD

**Issue by marumari** [mozilla/tls-observatory#128]
Some examples:

mozilla.org returns an A (87/100), SSL Labs gives it a C
pokeinthe.io returns an A (94/100), SSL Labs gives it an A+
unknownartists.co (randomly picked from their hall of shame) returns an A (90/100), SSL Labs gives it an F

**Comment by jvehent** [comment]
Somewhat related, I find the grading a little too nice. For example, `usairways.com` with their horrible SSLV2 and RC4-MD5 gets a C. I think we'll need to implement the plateaus that SSLLabs has built in.

```
--- Ciphers Evaluation ---
prio cipher          protocols pfs  curves
1    AES128-SHA      TLSv1     None 
2    AES256-SHA      TLSv1     None 
3    RC2-CBC-MD5     SSLv2     None 
4    RC4-MD5         SSLv2     None 
5    DES-CBC3-MD5    SSLv2     None 
6    RC4-64-MD5      SSLv2     None 
7    DES-CBC-MD5     SSLv2     None 
8    EXP-RC2-CBC-MD5 SSLv2     None 
9    EXP-RC4-MD5     SSLv2     None 

--- Analyzers ---
* Grade: C (63/100)
```

**Comment by 0xdiba** [comment]
Yes grade caps are definitely the next thing we should implement.
Right now the analyser only takes into account the key lengths and protocol
"age".
Although SSLv2 support should have resulted in a bad score even now...
I'm going to look into that first.

**Comment by floatingatoll** [comment]
Ping. Is this endpoint still suffering the above-described issues, and/or @april do you have this data through other more valid means?

**Comment by april** [comment]
As near as I can tell, the TLS Observatory grading is still broken or at least doesn't anywhere closely match what SSL Labs produces.
